### ISSUE TITLE
[Bug]: every time I try to run a sphericity test on a repeated measures anova the whole program stops responding - for days! 

### ISSUE THREAD

**Issue by ataylor92110** [jasp-stats/jasp-issues#2229]
### JASP Version

0.17.2.1

### JASP Module

ANOVA

### What analysis are you seeing the problem on?

Repeated Measure Anova (with one between subjects factor)

### What OS are you seeing the problem on?

Windows 11

### Bug Description

I was not able to run sphericity tests for a manuscript before submitting it because it kept locking up the program so I just submitted the ms. But now the reviewers want the sphericity tests! Same program. I do have a screen shot but it doesn't show much. I hope the paste from the printscreen function worked. Also, I cannot get a compressed file because it's frozen right now. I did leave it working over night but this morning same thing. This is a 3 level repeated measure with a 2-level between factor so it's a 2x3 RM ANOVA. This does not seem like a complicated analysis to cause this much trouble but the file is already huge. At this point I'd be happy to split it into two but I don't know how to do that. 
![image](https://github.com/jasp-stats/jasp-issues/assets/55157337/56a4b31d-0658-4b7f-a245-16d4097e862f)

### Expected Behaviour

I should have received the prior analysis without sphericity tests now reanalyzed with sphericity tests. 

### Steps to Reproduce

1. Run anova without sphericity tests
2. Run anova with sphericity tests
3.
...

### Log (if any)

I tried to upload this but now JASP won't load at all. I would be willing to try to send whatever files I have if you can direct me to the proper ones to send you. This is not my first time with this issue and I wonder why it repeats after being fixed. 

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if possible.
- [ ] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by RensDofferhoff** [comment]
I fear I cannot reproduce this bug.
Could you provide a jasp file or more detailed description of steps?
How many rows does the data file have approximately?

**Comment by ataylor92110** [comment]
Well, I fixed it for now. I think the file was TOO large ;-) I broke it
down into Anovas and all other analyses. It now works. I "down-sized" my
laptop last time I had to replace it and I think this one has far less
computing power. Now that I am sort of retired (haha, no such thing for
true academics but I knew when it was time to leave the formal academy), I
thought I could do with a "smaller" one. So as long as I keep my files
reasonable - it's still not small with about a dozen Anovas in there, but
it's manageable - I think I will be OK. I think I sought help in
frustration.  It did eventually freeze my entire laptop yesterday and not
just the program but once I got it running again, the first thing I did was
split that file and so far, so good. So your question about lines is
probably right on. And because of the program freezing I wasn't able to
include the logs and such in the original help request. That is a catch-22.

Thanks for the prompt reply. This is what makes JASP great, especially for
those on a tighter budget these days.

Annette
\
Annette Kujawski Taylor, Ph.D.

she/her/hers ("What Are Personal Pronouns & Why Do They Matter?"
<https://www.mypronouns.org/what-and-why>)
Professor, Emerita, Psychological Sciences
University of San Diego
***@***.***
